CUDA.Dockerfile

```diff
@@ -1,10 +1,8 @@
-# Build argumnets
-ARG CUDA_VER=11.8.0
-ARG UBUNTU_VER=22.04
```

Comment:
I have removed these ARGs. We can only build with 11.8.0, and aihwkit does not build with CUDA 12.x versions.

Reply:
Edit: It builds fine, but while running requires some `cudablas11.so` files that are nit available with CUDA 12.1.

Reply:
I will update

Reply:
pushed a fix